wallet-sdk/src/test/kotlin/org/stellar/walletsdk/AnchorTest.kt

```diff
@@ -77,18 +78,18 @@ internal class AnchorTest {
     fun `get interactive deposit URL with different funds account`() {
       val depositResponse = runBlocking {
         val authToken =
-          anchor.getAuthToken(
+          anchor
+            .auth(toml = toml, walletSigner = InProcessWalletSigner())
+            .authenticate(ADDRESS_ACTIVE)
+
+        anchor
+          .interactive()
+          .deposit(
             accountAddress = ADDRESS_ACTIVE,
-            toml = toml,
-            walletSigner = InProcessWalletSigner()
+            fundsAccountAddress = ADDRESS_ACTIVE_TWO,
+            assetCode = ANCHOR_ASSET_CODE,
+            authToken = authToken,
           )
```

Comment:
Looking at how this is used, it might be convenient to have params passed to `interactive(...)`, then `deposit` or `withdraw` can be called without any params (they are the same for both of them). Thoughts?

Reply:
Yes, but user can won't be able to re-use Interactive object in that case